tensorflow/core/profiler/utils/hardware_type_utils.cc

```diff
@@ -28,7 +29,8 @@ namespace {
 // cycle per streaming multiprocessor.
 // https://docs.nvidia.com/cuda/cuda-c-programming-guide/index.html#arithmetic-instructions__throughput-native-arithmetic-instructions
 uint32 GetFmaMaxThroughputPerSMPerCycle(const DeviceCapabilities& device_cap) {
-  if (device_cap.dev_manufacturer() == "Nvidia") {
+  if (device_cap.device_manufacturer() ==
+      tensorflow::tfprof::kDeviceManufacturerNvidia) {
```

Comment:
tensorflow::tfprof is a deprecated namespace / codebase.
please use tensorflow::profiler

Reply:
Done.